### PR TITLE
BinSearch for NumItemsBetween and ItemsBetween

### PR DESCRIPTION
**Pull request by prateshg**

### CODE REVIEW

**Comment by akshayknarayan** [comment]
Sorry @prateshg, missed this - will review now

File: history.go [hunk 1]
```diff
@@ -91,28 +119,29 @@ func binsearch(wanted time.Time, times []time.Time) time.Time {
 
 func linsearch(wanted time.Time, times []time.Time) time.Time {
 	then := times[0]
+	count := 0
 	for _, t := range times {
 		if t.After(wanted) {
 			return then
 		} else {
 			then = t
+			count += 1
```
Comment (akshayknarayan) [inline history.go]: is this variable used?

**Review by akshayknarayan — CHANGES_REQUESTED** [review]
Overall looks good. 2 minor changes:

- squash the 3 commits into one commit
- remove unused variable "count" from `linsearch()` fn in `history.go`

**Comment by akshayknarayan** [comment]
Ah, sorry, you removed it in a later commit.

Just squash the commits into one commit and I'll merge it.

**Comment by prateshg** [comment]
I am gonna close this pull request and create a new one with squashed commits

**Comment by akshayknarayan** [comment]
you don't have to create a new PR - just rebase, squash, and force push to the branch.

**Comment by prateshg** [comment]
Done

**Pull request merged**